tests/cargo-kani/simple-extern/src/lib.rs

```diff
@@ -2,6 +2,8 @@
 // SPDX-License-Identifier: Apache-2.0 OR MIT
 pub mod externs;
 pub use externs::external_c_assertion;
+// TODO: our reachability analysis does not see through C functions
+pub use externs::rust_add1;
```

Comment:
Should we track this in a GitHub issue?

Reply:
https://github.com/model-checking/kani/issues/3263